client/lib/mixins/emitter/index.js

```diff
@@ -5,16 +5,7 @@
  */
 
 import { assign } from 'lodash';
-
-// Here we make sure we’re not using Node’s core events module,
-// because it contains a `domain` property and it clashes with our
-// `Site` module that also has the same property.
-//
-// By adding a slash we force `require` to go into directory mode and to
-// load the module from `node_modules/` instead of the core’s `events.js`
-// file. Webpack uses the same module on the client side, too, which
-// makes for a nice consistency.
-import { EventEmitter } from 'events/';
+import { EventEmitter } from 'events';
```

Comment:
Did you verify somehow that the `domain` property on `EventEmitter`-decorated objects is no longer a threat? The `Site` class mentioned in the comment is gone, but there might be other usages that are still live.

Reply:
I'll be frank:  I don't understand the comments or how it could make sense.
`webpack` never bundled the actual "node module".  It always used a web polyfill.  In this specific case it was [Gozala/events](https://github.com/Gozala/events) which is published to npm as `events`.

If you look at the [source](https://github.com/Gozala/events/blob/master/events.js), you won't find `domain` anywhere.

Reply:
discussed on slack. I think the comment was largely historical based on where this file used to live in the pre-oss days